encoding/protobuf/error.go

```diff
@@ -83,20 +83,13 @@ func GetErrorDetails(err error) []proto.Message {
 	return nil
 }
 
-func IsProtobufError(err error) bool {
-	_, ok := err.(*pberror)
-	return ok
-}
-
 // ErrorOption is an option for the NewError constructor.
 type ErrorOption struct{ apply func(*pberror) }
 
-// WithDetails sets the details of the error.
-func WithDetails(details ...proto.Message) ErrorOption {
+// WithErrorDetails sets the details of the error.
```

Comment:
sets -> appends/adds

Reply:
still, docstring does not match behavior.